src/main/webapp/features.jsp

```diff
@@ -2,187 +2,216 @@
+                <img class="features-page-img" src="images/raised-edge.png">
             </div>
         </section>
 
         <section class="row">
             <div class="col-xs-12 col-sm-8 col-sm-offset-2">
                 <h2 class="color_orange col-xs-12">Powerful search</h2>
-                <img class="img-responsive col-xs-12" src="images/feature-powerfulsearch.png" alt="Powerful search">
-                <div class="col-xs-12 col-sm-10 col-sm-offset-1">
-                    TEAMMATES has a powerful search feature to locate details about students or past comments.
-                    <br><br>
-                    Given above is an example of searching for a student whose name you are not very sure of.
+                <img class="features-page-img" src="images/feature-powerfulsearch.png" alt="Powerful search"><div class="col-xs-12 col-sm-9">
+                    <div class="features-page-text">
+                        TEAMMATES has a powerful search feature to locate details about students or past comments.
+                        <br><br>
+                        Given above is an example of searching for a student whose name you are not very sure of.
+                    </div>
                 </div>
-                <img class="img-responsive" src="images/raised-edge.png">
+                <img class="features-page-img" src="images/raised-edge.png">
             </div>
         </section>
 
         <section class="row">
             <div class="col-xs-12 col-sm-8 col-sm-offset-2">
                 <h2 class="color_orange col-xs-12">Support for big courses</h2>
-                <img class="img-responsive col-xs-12" src="images/feature-supportforbigcourses.png" alt="Support for big courses">
-                <div class="col-xs-12 col-sm-10 col-sm-offset-1">
-                    TEAMMATES can support a course of any size (even beyond 1,000 students), as long as you divide the students into sections of no more than 100 students.
-                    There is no limit on the number of courses or sessions you could have either.
+                <img class="features-page-img" src="images/feature-supportforbigcourses.png" alt="Support for big courses">
+                <div class="col-xs-12 col-sm-9">
+                    <div class="features-page-text">
+                        TEAMMATES can support a course of any size (even beyond 1,000 students), as long as you divide the students into sections of no more than 100 students.
+                        There is no limit on the number of courses or sessions you could have either.
+                    </div>
                 </div>
-                <img class="row img-responsive" src="images/raised-edge.png">
+                <img class="row features-page-img" src="images/raised-edge.png">
             </div>
         </section>
 
         <div class="row">
             <p class="col-xs-12 col-sm-8 col-sm-offset-2">
+            <br>
```

Comment:
HTML is meant to organise webpage content. Adding additional `<br>` tags to introduce a gap of whitespace (for purposes other than formatting text into different lines) is discouraged (see #3866 for an explanation).